data/antivirus.yaml

```diff
@@ -26,8 +32,12 @@ sources:
 - type: FILE
   attributes:
     paths:
-    - '%%environ_allusersappdata%%\Microsoft\Windows Defender\Support\MPLog-*.log'
+    - '%%environ_allusersappdata%%\Microsoft\Windows AntiMalware\Support\MPDetection-*.log'
+    - '%%environ_allusersappdata%%\Microsoft\Windows AntiMalware\Support\MPLog-*.log'
+    - '%%environ_allusersappdata%%\Microsoft\Windows Defender\Scans\History\Service\DetectionHistory\**'
     - '%%environ_allusersappdata%%\Microsoft\Windows Defender\Support\MPDetection-*.log'
+    - '%%environ_allusersappdata%%\Microsoft\Windows Defender\Support\MPLog-*.log'
+    - '%%environ_systemroot%%\Temp\MpCmdRun.log'
```

Comment:
is this correct? or should this be `%%users.temp%%` ?

Reply:
According to the source, it was C:\Windows\Temp, and not %%users.temp%%.

Reply:
Are you sure it is not using %Temp% and C:\Windows\Temp was just a single observation of this ?

Reply:
I just did some research and it appears that in older versions of Windows [1][2] it is indeed C:\Windows\Temp and more recently it became %Temp%. It's also possible that the file would reside in C:\Windows\ServiceProfiles\LocalService\AppData\Local\Temp [3]. I'll add those to the definition to cover all 3 variants.

[1] https://www.tenforums.com/general-support/165610-location-mpcmdrun-log.html
[2] https://www.tenforums.com/general-support/165610-location-mpcmdrun-log.html